pyemma/coordinates/transform/transformer.py

```diff
@@ -176,6 +176,12 @@ def describe(self):
         """ get a representation of this Transformer"""
         return self.__str__()
 
+    def dimension(self):
```

Comment:
In my opinion we should raise NotImplError here, since it might lead to strange errors for dim=0 in subsequent usage of this.